src/handlers/payout/index.ts

```diff
@@ -0,0 +1,44 @@
+import { getWalletAddress } from "../../adapters/supabase";
+import { getBotContext } from "../../bindings";
+import { addCommentToIssue, generatePermit2Signature } from "../../helpers";
+import { Payload } from "../../types";
+import { bountyInfo } from "../wildcard";
+
+export const handleIssueClosed = async () => {
+  const context = getBotContext();
+  const { log } = context;
+  const payload = context.payload as Payload;
+  const issue = payload.issue;
+  if (!issue) return;
+
+  log.info(`Handling issues.closed event, issue: ${issue.number}`);
+  const issueDetailed = bountyInfo(issue);
+  if (!issueDetailed.isBounty) {
+    log.info(`Skipping... its not a bounty`);
+    return;
+  }
+
+  const assignees = issue?.assignees ?? [];
+  const assignee = assignees.length > 0 ? assignees[0] : undefined;
+  if (!assignee) {
+    log.info("Skipping to proceed the payment because `assignee` is undefined");
+    return;
+  }
+
+  if (!issueDetailed.priceLabel) {
+    log.info("Skipping to proceed the payment because price not set");
+    return;
+  }
+
+  const priceInETH = issueDetailed.priceLabel!.substring(7, issueDetailed.priceLabel!.length - 4);
+  const recipient = await getWalletAddress(assignee);
+  if (!recipient) {
+    log.info(`Recipient address is missing`);
+    await addCommentToIssue(`@${assignee} would you please post your wallet address here?`, issue.number);
+    return;
+  }
+
+  const payoutUrl = await generatePermit2Signature(recipient, priceInETH);
+  log.info(`Posing a payout url to the issue, url: ${payoutUrl}`);
+  await addCommentToIssue(`@${assignee} You can receive your payment on ${payoutUrl}`, issue.number);
```

Comment:
```suggestion
  await addCommentToIssue(`@${assignee} [ [ CLAIM ${CLAIM_AMOUNT} ${CLAIM_TOKEN_SYMBOL} ] ](https://pay.ubq.fi/?claim=${ENCODED_CLAIM})`, issue.number);
```

Reply:
Regarding that security related point, we could consider posting their address as part of this comment as well. It could be even faster/easier to notice something is wrong. 
---
Another idea (not great) is that we could consider only supporting ENS and only allowing unique ENS names to be registered. That would make it very tedious for an attacker (but also kind of annoying for bounty hunters)

Reply:
https://github.com/ubiquity/bounty-bot/pull/138/commits/838ada446e5223d63893a4927f5d6ad0e7d4377b

Reply:
Thanks! Please also add their wallet address to the comment!

Reply:
https://github.com/ubiquity/bounty-bot/pull/138/commits/1db28cd3f74f502ce9f5c6823c285bc1bf1fedfe